content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
+Doing so enables the ability to create alternative |RfQs|, as well as *blanket orders*.
 
-.. image:: calls_for_tenders/calls-for-tenders-settings-page.png
+.. image:: calls_for_tenders/calls-for-tenders-enabled-setting.png
    :align: center
    :alt: Purchase Agreements enabled in the Purchase app settings.
 
 .. tip::
-   To save time on a *call for tender*, custom vendors, prices, and delivery lead times can be set
-   in the :guilabel:`Purchase` tab on a product form. To do so, navigate to
-   :menuselection:`Purchase --> Products --> Products`, and select a product to edit. From the
-   product form, click the :guilabel:`Purchase tab`, then click :guilabel:`Add a line`. From the
-   drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, and set a
-   :guilabel:`Price` and :guilabel:`Delivery Lead Time` if desired. Clicking the
-   :guilabel:`additional options (two-dots)` icon provides additional visibility options to add to
-   the line item.
-
-Create an :abbr:`RFQ (Request for Quotation)`
-=============================================
-
-To create a new :abbr:`RFQ (Request for Quotation)`, navigate to the :menuselection:`Purchase` app,
-and click :guilabel:`New`.
-
-Then, add information to the :abbr:`RFQ (Request for Quotation)` form: add a vendor from the
-drop-down next to the :guilabel:`Vendor` field, and click :guilabel:`Add a product` to select a
-product from the drop-down menu in the :guilabel:`Product` column. Then, set the desired purchase
-quantity in the :guilabel:`Quantity` column, and change the purchase price in the :guilabel:`Unit
-Price` column, if desired.
-
-Clicking the :guilabel:`additional options (two-dots)` icon provides additional visibility options
-to add to the line item. Repeat these steps to add as many options as desired, including the
-:guilabel:`UoM` (Units of Measure) to purchase the products in, and the :guilabel:`Expected Arrival`
-date.
-
-Once ready, click :guilabel:`Send by Email`. This causes a :guilabel:`Compose Email` pop-up window
-to appear, wherein the message to the vendor can be customized. Once ready, click :guilabel:`Send`.
-This turns the :abbr:`RFQ (Request for Quotation)` into a purchase order (PO), and sends an email
-to the vendor listed on the purchase order form.
-
-.. image:: calls_for_tenders/calls-for-tenders-email-popup.png
+   To save time creating alternative |RfQs|, custom vendors, prices, and delivery lead times can be
+   set in the :guilabel:`Purchase` tab of a product form.
+
+   Navigate to :menuselection:`Purchase app --> Products --> Products`, and select a product to
+   edit. On the product form, click the :guilabel:`Purchase` tab, and click :guilabel:`Add a line`.
+
+   From the drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, and set a
+   :guilabel:`Price` and :guilabel:`Delivery Lead Time`, if necessary. Clicking the
+   :guilabel:`⋮ (three dots)` icon provides additional visibility options to add to the line item.
```

Comment:
I'm gonna go with :guilabel:`(optional columns drop-down toggle) icon` - because that's what's showing in the Inspect code... I know it's wordy 😕 If I need to change it again I will...

![optional columns icon](https://github.com/odoo/documentation/assets/120495567/db43b776-4524-4fe8-9bcc-d689822ca704)